src/dmd/mars.d

```diff
@@ -2146,9 +2149,32 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
                 goto Lerror;
             }
         }
+	else if (p[1] == 'H' && p[2] == 'c') // https://dlang.org/dmd.html#switch-Hc
+	{
+            params.doCHdrGeneration = true;
+
+            switch (p[3])
+            {
+            case 'd':               // https://dlang.org/dmd.html#switch-Hcd
+                if (!p[4])
+                    goto Lnoarg;
+                params.chdrdir = (p + 4 + (p[4] == '=')).toDString;
+                break;
+            case 'f':               // https://dlang.org/dmd.html#switch-Hcf
+                if (!p[4])
+                    goto Lnoarg;
+                params.chdrname = (p + 4 + (p[4] == '=')).toDString;
+                break;
+            case 0:
+                break;
+            default:
+                goto Lerror;
+            }
+	}
```

Comment:
It seems you have a number of tabs here which the CI doesn't like

Reply:
Oops.  I assumed all of the failures were because of output format changes.  Fixed.